blog/2024-07-30-how-to-run-webassembly-components-on-kubernetes-with-wasmcloud/index.mdx

```diff
@@ -0,0 +1,178 @@
+
+In this tutorial, you'll learn how to get started running WebAssembly components on Kubernetes in **three simple steps**. The whole process should only take about five minutes.
+
+{/* truncate */}
+
+### How does wasmCloud work with Kubernetes?
+
+One way to think of wasmCloud is as **Kubernetes for WebAssembly**. We sometimes describe it as a "**Wasm-native orchestrator**": a distributed applications platform built from the ground up for WebAssembly workloads.  
+
+Because wasmCloud is tailored specifically to WebAssembly, it can take advantage of components' distinctive efficiency, interoperability, and composability. While you can run wasmCloud standalone, most teams don't want to replace their entire infrastructure. Kubernetes is very good at abstracting and managing the infrastructure side of the equation&mdash;wasmCloud sits on that foundation and focuses on orchestrating distributed applications. 
+
+On a technical level, wasmCloud integrates with Kubernetes through an [operator](https://github.com/wasmCloud/wasmcloud-operator). Once you install the operator on your Kubernetes cluster, you can...
+
+* **Deploy [wasmCloud hosts](/docs/concepts/hosts) on the cluster**. At a high level, you can think of a host as a runtime environment that loads, runs, and communicates between the different parts of your application.
+* **Deploy wasmCloud applications to the cluster straight from `kubectl`**. In wasmCloud, [applications are defined in YAML manifests](../../docs/ecosystem/wadm/model) with the [OAM specification](https://oam.dev/). The manifests will look familiar to Kubernetes users&mdash;better yet, you can deploy them to the cluster the same way you would any other Kubernetes resource. 
+
+What does all of this mean in practice?
+
+1. Developers can move faster, reusing building blocks written against well-known APIs. 
+2. It's easy to run distributed apps in resource-constrained environments at the edge. 
+3. You can use wasmCloud's connectivity layer to connect Kubernetes clusters across different clouds or regions.
+4. Workloads can be scheduled dynamically across regions and clouds. 
+
+By unlocking the power of distributed components, wasmCloud extends Kubernetes as well. Let's jump in and see how it works.
+
+## Get started with wasmCloud on Kubernetes
+
+In this tutorial, we'll get wasmCloud up and running on a local Kubernetes cluster. You'll need [Helm](https://helm.sh/) and a Kubernetes cluster running on your machine. It also helps to install [wasmCloud Shell (`wash`)](/docs/installation), though it's not strictly necessary.
+
+:::info[Your friendly neighborhood Kubernetes]
+Make sure you have a Kubernetes cluster running locally. Some good options include [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), [k3s](https://k3s.io/), or [k0s](https://k0sproject.io/).
```

Comment:
```suggestion
Make sure you have a Kubernetes cluster running and API access to run `helm` and `kubectl` commands. Some good local options include [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), [k3s](https://k3s.io/), or [k0s](https://k0sproject.io/).
```

Just a suggestion in case folks want to run this in the cloud!